### PR TITLE
[PYTHON_BINDINGS] Expose navmesh triangles to Python

### PR DESCRIPTION
**Pull request by Skylion007**
## Motivation and Context

It allows Python code to directly get the triangles from the NavMesh, before it was impossible to get NavMesh data into Python directly. This will add a property to pathfinders that is an array of 3D arrays indicating the triangles.

## How Has This Been Tested
Locally, it builds properly 

## Types of changes

- [X] New feature (non-breaking change which adds functionality)

## Checklist

- [X] My code follows the code style of this project.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/esp/bindings/ShortestPathBindings.cpp [hunk 1]
```diff
@@ -95,6 +96,12 @@ void initShortestPathBindings(py::module& m) {
       .def("island_radius", &PathFinder::islandRadius, "pt"_a)
       .def_property_readonly("is_loaded", &PathFinder::isLoaded)
       .def_property_readonly("navigable_area", &PathFinder::getNavigableArea)
+      .def_property_readonly(
```
Comment (erikwijmans) [inline src/esp/bindings/ShortestPathBindings.cpp:99]: These should probably be methods to discourage them from being called in a tight loop

Comment (Skylion007) [reply src/esp/bindings/ShortestPathBindings.cpp:99]: Haven't heard that argument before... we could cache it potentially?

Comment (erikwijmans) [reply src/esp/bindings/ShortestPathBindings.cpp:99]: The general expectation of properties IMO is that they are lightweight,

```python
for vid in vids:
    pathfinder.navmesh_vertices[vid]
```

seems fine while
```python
for vid in vids:
    pathfinder.build_navmesh_vertices()[vid]
```

definitely doesn't.


Yeah, caching would be good.

Comment (Skylion007) [reply src/esp/bindings/ShortestPathBindings.cpp]: @erikwijmans Done

**Review by erikwijmans — APPROVED** [review]

**Pull request merged**